### PR TITLE
PFM-1192 : Works inbox config

### PR DESCRIPTION
**Pull request by pintu-eGov**

### CODE REVIEW

File: deploy-as-code/helm/charts/digit-works/backend/works-inbox-service/values.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  - name: JAVA_OPTS
+    value: {{ index .Values "heap" | quote }}
+  - name: JAVA_ARGS
+    value: {{ index .Values "java-args" | quote }}
+  {{- if index .Values "tracing-enabled" }}
+  - name: TRACER_OPENTRACING_ENABLED
+    value: "true" 
+  {{- end }}
+  - name: SERVER_CONTEXT_PATH
+    value: /inbox
```
Comment (rushang7-eGov) [inline deploy-as-code/helm/charts/digit-works/backend/works-inbox-service/values.yaml:41]: Wouldn't it be "/works-inbox-service"? 
Same for the line below.

File: deploy-as-code/helm/environments/works-dev.yaml [hunk 2]
```diff
@@ -182,6 +182,13 @@ estimate-service:
 loi-service:
   tracing-enabled: true
 
+works-inbox-service:
+  tracing-enabled: true
+  es-search-pagination-default-limit: 50
```
Comment (rushang7-eGov) [inline deploy-as-code/helm/environments/works-dev.yaml:187]: If we are not using ES, then remove it from the config.

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

File: deploy-as-code/helm/charts/digit-works/backend/works-inbox-service/values.yaml (hunk 1 above)
Comment (pintu-eGov) [reply deploy-as-code/helm/charts/digit-works/backend/works-inbox-service/values.yaml:41]: I thought of keeping the same context path as we have in other municipal services. if you want it be "/works-inbox-service", I'll change them.

File: deploy-as-code/helm/environments/works-dev.yaml (hunk 2 above)
Comment (pintu-eGov) [reply deploy-as-code/helm/environments/works-dev.yaml:187]: will discuss on this...

File: deploy-as-code/helm/charts/digit-works/backend/works-inbox-service/values.yaml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  - name: ES_SEARCH_PAGINATION_MAX_SEARCH_LIMIT
+    value: {{ index .Values "es-search-pagination-max-search-limit" | quote }} 
+  {{- end }}
+  {{- if index .Values "es-search-default-sort-order" }}
+  - name: ES_SEARCH_DEFAULT_SORT_ORDER
+    value: {{ index .Values "es-search-default-sort-order" | quote }} 
+  {{- end }}
+  {{- if index .Values "state-level-tenant-id" }}
+  - name: STATE_LEVEL_TENANT_ID
+    value: {{ index .Values "state-level-tenant-id" | quote }}
```
Comment (rushang7-eGov) [inline deploy-as-code/helm/charts/digit-works/backend/works-inbox-service/values.yaml:72]: Don't read it from there. Either remove it of not required, or read it from the config map.

File: deploy-as-code/helm/environments/works-dev.yaml [hunk 4]
```diff
@@ -182,6 +182,10 @@ estimate-service:
 loi-service:
   tracing-enabled: true
 
+works-inbox-service:
+  tracing-enabled: true
+  state-level-tenant-id: pb
```
Comment (rushang7-eGov) [inline deploy-as-code/helm/environments/works-dev.yaml:187]: Remove this.

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

File: deploy-as-code/helm/environments/works-dev.yaml (hunk 4 above)
Comment (pintu-eGov) [reply deploy-as-code/helm/environments/works-dev.yaml:187]: done

File: deploy-as-code/helm/environments/works-dev.yaml [hunk 5]
```diff
@@ -60,6 +60,7 @@ cluster-configs:
         timezone: "Asia/Kolkata"
         es-infra-host: elasticsearch-data-v1.es-cluster
         db-ssl: "false"
+        state-level-tenant-id: pb
```
Comment (rushang7-eGov) [inline deploy-as-code/helm/environments/works-dev.yaml:63]: Don't add it. It is already present, look above

**Review by rushang7-eGov — APPROVED** [review]

**Pull request merged**